NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -122,20 +122,6 @@ public class NightscoutUploader {
         uploadToNS(treatments, endpoint: defaultNightscoutTreatmentPath, completion: completionHandler)
     }
 
-    public func getPumpStatusFromMySentryPumpStatus(status: MySentryPumpStatusMessageBody) throws -> PumpStatus {
```

Comment:
That was just the logic to convert MySentryPumpStatusMessageBody into PumpStatus.  I moved it into DDM, as it felt out of place in the uploader (really uses no state of the uploader).  https://github.com/ps2/rileylink_ios/blob/dev/RileyLink/DeviceDataManager.swift#L212-L226

Looking at that, I see I forgot to convert the throw.

Where would you suggest logic like this resides?  While I don't want to clutter up DDM too much, things that bridge frameworks (like this method bridges MinimedKit and NightscoutUploadKit models) seem to be ending up here (in DDM) at the high level, and that at least does have the advantage of keeping the frameworks focused.